src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Location/style.scss

```diff
@@ -26,8 +26,8 @@
     }
 
     #mapdiv {
-      height: 100px !important;
-      width: 100px !important;
+      height: 80px !important;
+      width: 80px !important;
```

Comment:
Ugh, important is pretty nasty. Especially on a #id selector.

Reply:
I'm getting issues where multiple maps on the same page break because we are using identifiers to style (and select). Lets try and use classes where possible.